test/components/enketo-iframe.spec.js

```diff
@@ -77,6 +77,33 @@ describe('EnketoIframe', () => {
     wrapper.vm.$router.push.calledWith('/projects/1').should.be.true;
   });
 
+  it('redirects on submissionsuccess message with return_url - external', async () => {
+    const fakeAssign = sinon.fake();
+    const wrapper = mountComponent({
+      props: { enketoId, actionType: '', instanceId: 'test-instance' },
+      container: {
+        router: mockRouter('/?return_url=http://example.com/projects/1'),
+        location: {
+          origin: window.location.origin,
+          assign: (url) => {
+            fakeAssign(url);
+          }
+        }
+      }
+    });
+    const iframe = wrapper.find('iframe');
+
+    const script = document.createElement('script');
+    script.textContent = `
+      window.parent.postMessage(JSON.stringify({ enketoEvent: 'submissionsuccess' }), "*");
+    `;
+    iframe.element.contentDocument.body.appendChild(script);
```

Comment:
This is just a few lines, but I feel like it could help to move it into a little utility function that's housed in this file. Then the test would be shorter and easy to scan quickly.